src/hg_connect_http.rs

```diff
@@ -294,6 +294,9 @@ unsafe extern "C" fn trace_log_callback(
         CURLINFO_HEADER_OUT => logging::Direction::Send,
         _ => return 0,
     };
+
+    // c_char can either be u8 or i8 depending on each platform
+    #[allow(clippy::unnecessary_cast)]
```

Comment:
Clippy is actually happy with replacing `as *const u8` with `as *const _` instead.